Scraper/webScraper.py

```diff
@@ -0,0 +1,22 @@
+from bs4 import BeautifulSoup
+import requests
+
+# Save site url in a variable
+URL = "http://smashboards.com/threads/captain-falcon-hitboxes-and-frame-data.284165/"
+
+# Open the url and save the html data into bs4
```

Comment:
Needs a more clear 1-2 line description. Good questions to ask yourself when making the description: What input does this take? What does this function return? Does this function produce any noteworthy side effects? What is the main goal of this function?

e.g. 
**# Takes a URL and returns the bs4 webpage for that URL.**